diff --git a/text/english/dialog/vcchead.msg b/text/english/dialog/vcchead.msg
index f8dc4d8..ca75746 100644
--- a/text/english/dialog/vcchead.msg
+++ b/text/english/dialog/vcchead.msg
@@ -6,17 +6,17 @@
 {111}{}{Me? You must be mistaken. Who are you people?}
 {112}{}{I'm asking the questions here! I am Colonel Knight of the United States Government Armed Forces. Now I'm in charge. Your actions in the wasteland, and especially in the government vault, have caused quite a stir. We'll have to do a lot of work to fix the situation.}
 {113}{}{A colonel... What an uncommon thing these days.}
-{114}{}{[Contemptuously] You monkey... How such a pitiful thing like you could make so much trouble? You weren't created for this. Ha, even at the base everyone believes their "health insurance" has no ability to comprehend their existence, let alone speak and think for themselves.} 
+{114}{}{[Contemptuously] You monkey... How could such a pitiful thing like you make so much trouble? You weren't created for this. Ha, even at the base everyone believes their "health insurance" has no ability to comprehend their existence, let alone speak and think for themselves.}
 {115}{}{"Health insurance"? Is this how you call us?}
 
-{116}{}{Yes, we do, and that's what you are. Still have trouble accepting it? That's a shame. I was beginning to hope that you would say something meaningful, but it seems that your brain is too small for that. }
+{116}{}{Yes, we do, and that's what you are. Still have trouble accepting it? That's a shame. I was beginning to hope that you would say something meaningful, but it seems that your brain is too small for that.}
 {117}{}{What do you want?}
-{118}{}{Right, let's get down to business. We know that Bayham tried to get his hands on the ACS component, but was unable to do it, thanks to you. At one hand, I'm grateful. We couldn't handle the son of a bitch, despite him having a damn high opinion of himself. If he had seized the control of the cloning facility, Nevada would have become his playground.}
+{118}{}{Right, let's get down to business. We know that Bayham tried to get his hands on the ACS component, but was unable to do it, thanks to you. On the one hand, I'm grateful. We couldn't handle the son of a bitch, despite him having a damn high opinion of himself. If he had seized the control of the cloning facility, Nevada would have become his playground.}
 {119}{}{And?}
 {120}{}{But now he's out of the picture, you're here and you can return the component where it belongs. Hand it over. You don't have a choice anyway.}
 {121}{}{There's always a choice. I can destroy the device and the whole base with it and stop the cloning process forever.}
 {122}{}{I would anyway, but what are you going to do?}
-{123}{}{Destroy? Ha! Your brain is not suited to handle such a complicated task. The device that you're hiding is basically a hard drive structurally based on refractory crystals. Do you even UNDERSTAND what REFRACTORY means? In order to destroy it, you need a temperature level you can get only in a lab with proper equipment. Just hand it over - and everything will be fine.}
+{123}{}{Destroy? Ha! Your brain is not suited to handle such a complicated task. The device that you're hiding is a hard drive structurally based on refractory crystals. Do you even UNDERSTAND what REFRACTORY means? In order to destroy it, you need a temperature level you can get only in a lab with proper equipment. Just hand it over - and everything will be fine.}
 {124}{}{All other subjects like you have been isolated, as they should have been from the start. We will leave some of our personnel here to control the situation. Your Overseer is dead, which means that the deal we had with him is off... It was just a bad joke, anyway. Discipline and order - that is the thing most needed in your little underdeveloped zoo. And that's the way it's going to be from now on.}
 {125}{}{I don't have the component.}
 {126}{}{Fine, I'll bring the component.}
@@ -26,7 +26,7 @@
 {130}{}{[Leave]}
 {131}{}{You call yourself a colonel, but you look and talk like a hillbilly in the bar.}
 
-{160}{}{Bring me that device, before I completely lose my patience.}
+{160}{}{Bring me that device before I completely lose my patience.}
 {161}{}{I have the device. Take it and leave me alone.}
 {162}{}{I brought you something better - a bullet in your stupid head. Take it and sign below!}
 {163}{}{I will do what you ask.}
diff --git a/text/english/dialog/vccoppsn.msg b/text/english/dialog/vccoppsn.msg
index 6f55679..71514d7 100644
--- a/text/english/dialog/vccoppsn.msg
+++ b/text/english/dialog/vccoppsn.msg
@@ -22,14 +22,14 @@
 {121}{}{Remind me where to find your friend Orris.}
 {122}{}{Usually he can be found near the "Hard Rock" cafe. Tell him that his old friend Murphy is in trouble.}
 
-{123}{}{It was like that: I won in the casino. Not a lot, but you still can have a really good party with that money. When I went outside, the cops kettled me off without any explanation. Then brought me here, then this smart-ass from the mayor's office appeared and outlined how much money I have in my pockets, and in which bills. He knew it for sure, I remember him near the cash register in the casino. He followed me! Now it turns out like I stole these money from him.}
+{123}{}{It was like that: I won in the casino. Not a lot, but enough for a really good party, right? When I went outside, the cops kettled me off without any explanation. Then brought me here, then this smart-ass from the mayor's office appeared and outlined how much money I have in my pockets, and in which bills. He knew it for sure, I remember him near the cash register in the casino. He followed me! Now it turns out like I stole these money from him.}
 {124}{}{Who is this "smart-ass from the mayor's office"?}
 {125}{}{But can't they identify you in the casino?}
 {126}{}{Why should I believe you?}
 {127}{}{I see. I have to go.}
 {128}{}{Some Irwin guy, an employee of the City Hall. Looks like many people here know him, but I don't. I'm from the suburbs, I don't hang out with the locals.}
 {129}{}{Surely they can. They write everything down in this ledger book, but the cops don't care, they tend to believe that guy from the mayor's office. And I am sure that casino won't stand up for me. It's more convenient for them to lick asses of the City Hall and the cops.}
-{130}{}{You? You shouldn't. Why would you care... But I'm not asking you to believe me! I'm asking to talk to Orris, he will help, he's an old friend. And you might have something from it, you understand?}
+{130}{}{You? You shouldn't. Why would you care... But I'm not asking you to believe me! I'm asking to talk to Orris, he will help, he's an old friend. And you might get something from it, you understand?}
 
 {131}{}{What's going on?!}
 {132}{}{Well, apparently, the police officer in the waiting room is no longer at his duty. You should be happy.}
diff --git a/text/english/dialog/vccoprpr.msg b/text/english/dialog/vccoprpr.msg
index e5156d1..2d27a33 100644
--- a/text/english/dialog/vccoprpr.msg
+++ b/text/english/dialog/vccoprpr.msg
@@ -7,12 +7,12 @@
 {112}{}{Nothing.}
 {113}{}{"Fellow hard worker"? Heh! Bad, as always. Do you see this big guy on the floor?}
 {114}{}{What is it, a "robo-cop"? Looks funny...}
-{115}{}{Yeah, it sure does. So much complicated stuff to fix just so he can stand somewhere for intimidation. I'd have better luck wearing this junk on me and scaring passersby myself.}
+{115}{}{Yeah, it sure does. So much complicated stuff to fix just so he can stand somewhere and be intimidating. I'd have better luck wearing this junk on me and scaring passersby myself.}
 {116}{}{Do you think there would be no benefit from it?}
 {117}{}{Well, with full ammunition and in good condition, it's certainly a real death machine. Maybe the chief could even use it in special operations. But all in all, this thing is designed to intimidate by its appearance.}
-{118}{}{Do you think it will be dangerous after activation?}
+{118}{}{Do you think activating it could be dangerous?}
 {119}{}{Who knows... Military models were not created for peaceful purposes, baby. They were built to kill! And this robot's been dormant for about 70 years, with no checkups. Luckily, I figured out to strip it of its ammunition and disable the lower servo motors so I wouldn't be killed on the spot when it was turned on. But the thing that scares me is that not many people can understand this level of robotics. So no one can tell what's really in that piece of iron's mind. We have to rely on testing and spend a lot of time studying blueprints. So much effort for one robot! Is it worth it? I'm not sure.}
-{120}{}{Does it have any disadvantages?}
+{120}{}{Does it have any weaknesses?}
 {121}{}{Of course! It's full of electronics. One shot of an impulse weapon and the circuits just fry. This is a common problem with all pre-war robots.}
 {122}{}{Pulse weapon? And where can one find it?}
 {123}{}{I guess only in some military depots or places associated with the old army. These things were not produced for the civilian market.}
diff --git a/text/english/dialog/vccopzam.msg b/text/english/dialog/vccopzam.msg
index 8729f15..786f02c 100644
--- a/text/english/dialog/vccopzam.msg
+++ b/text/english/dialog/vccopzam.msg
@@ -22,7 +22,7 @@
 {124}{}{No reason. Why?}
 {125}{}{Your boss has given me a civil job to snitch Phillips.}
 
-{126}{}{Oh, then this changes things. At first we have been contacted by anonymous individual and notified, that Phillips makes drugs in his cafe. We went at the site, and found there some of that yellow stinky crap. Our specialists still try to figure out what is it. But they are still struggling, so if they can't identify this substance, then I don't know, how all this is going to end.}
+{126}{}{Oh, then this changes things. At first we have been contacted by anonymous individual and notified that Phillips makes drugs in his cafe. We went at the site, and found there some of that yellow stinky crap. Our specialists still try to figure out what is it. But they are still struggling, so if they can't identify this substance, then I don't know, how all this is going to end.}
 {127}{}{Yellow substance? Like this? [show an emulsifier]}
 {128}{}{Hmm. Okay.}
 {129}{}{Then you'll have to hurry, because during the search in the cafe we found dubious yellow substance resembling components for the production of drugs. Most likely, he will be sent to prison.}
@@ -30,7 +30,7 @@
 {131}{}{Oh so it's your friend! Well then, guys, you'll have to say goodbye to each other. Because soon Sam Phillips will be behind bars in prison. End of story.}
 {132}{}{Damn it, is it really that bad?}
 {133}{}{Drugs?! I don't believe it! You're lying!}
-{134}{}{[Looks at you from head to bottom] Independent detective? How interesting. Well, here's how it is. We have been contacted by an anonymous individual, he said, that Sam Phillips makes drugs. We went to the cafe, searched it and found some of that yellow stinky crap there. We're not sure yet what it is, but most likely his guilt will be proved by that thing alone.}
+{134}{}{[Looks at you from head to bottom] Independent detective? How interesting. Well, here's how it is. We have been contacted by an anonymous individual, he said that Sam Phillips makes drugs. We went to the cafe, searched it and found some of that yellow stinky crap there. We're not sure yet what it is, but most likely his guilt will be proven by that thing alone.}
 {135}{}{Now I see. Thanks.}
 
 {136}{}{Yes. It is exactly the same thing we found at Phillips cafe! Where did you get that?}
@@ -41,7 +41,7 @@
 {141}{}{How interesting. So what is this drug?}
 {142}{}{Wow... Food supplement... It doesn't look like it. But... How would I know how food supplement looks like? So that's why our specialists can't figure this stuff out. Well, I think your expertise is enough to prove Sam Phillips innocence.}
 {143}{}{That's great! I'll go tell him.}
-{144}{}{You, of course, are right, and we will do that immediately. Phillips will be arrested, and you, as it is being done in such cases, will be rewarded. Here, 1000$. Thank you, civilian, for your cooperation. I hope you will continue to notify us about any crimes that may be present.}
+{144}{}{You, of course, are right, and we will do that immediately. Phillips will be arrested, and you, as it is being done in such cases, will be rewarded. Here, 1000$. Thank you, civilian, for your cooperation. I hope you will continue to notify us about any crimes that may be occurring.}
 {145}{}{Heh, easy money.}
 
 #Про Высера
@@ -54,13 +54,13 @@
 
 #Джей Кукиш
 {160}{}{I'm looking for jay Kukish, a biker. Heard anything about him?}
-{161}{}{Wow - what a coincidence. We are also looking for it! We have orders to detain the offender. If you meet him - remember, he is very dangerous. Do not play around - shoot first, ask questions later.}
+{161}{}{Wow - what a coincidence. We are also looking for him! We have orders to detain the offender. If you meet him - remember, he is very dangerous. Do not play around - shoot first, ask questions later.}
 {162}{}{Uh, thanks for the info.}
 {163}{}{Very well. It was a correct decision.}
 
 #Джей Кукиш
-{170}{}{I demand to release Murphy from the isolator.}
-{171}{}{This possibility is allowed in two cases: either you will prove his innocence, or pay a deposit of 500$.}
+{170}{}{I demand that you release Murphy from his cell.}
+{171}{}{You've got two options: either prove his innocence, or pay a deposit of 500$.}
 {172}{}{I will pay. Here's the money.}
 {173}{}{No use talking to you. See you.}
 {174}{}{Prisoner Murphy! You're free to go.}
@@ -68,15 +68,15 @@
 {176}{}{The money is more important than your health? And if you get infected?}
 {177}{}{Ha-ha-ha, you don't say? This trick is a hundred years old. I've heard everything: "Wasteland disease", "shakes", "Narg syndrome"... What did you come up with this time?}
 {178}{}{You're telling me that, the "Wasteland disease" is not real? I'm a doctor with many years of experience, we have seen the whole villages die from that. Just like in Tonopah or Mesquite. Have you ever been there?}
-{179}{}{Hmm. I heard something about Tonopah. Something about mad people, consuming their own flesh... Or it was really the whole village, which just died out there... But since you're a doctor, I have to believe you. First, write a receipt that you vouch for their diagnosis.}
+{179}{}{Hmm. I heard something about Tonopah. Something about mad people, consuming their own flesh... Or it was really the whole village, which just died out there... But since you're a doctor, I have to believe you. First, write a receipt that you vouch for the diagnosis.}
 
 #Квест архивариуса и уличный воришка
 {180}{}{Is there any work here for a private detective?}
-{181}{}{Usually we don't hire anyone and deal by ourselves. But there is a few cases, where your help could be appreciated.}
-{182}{}{Interesting. What is it?}
+{181}{}{Usually we don't hire anyone and work by ourselves. But there are two cases where your help would be appreciated.}
+{182}{}{Interesting. Can you say more?}
 {183}{}{First, the daughter of the mayor's office archivist went missing. We haven't been able to find her. You can ask the archivist himself about his daughter. He is located on the second floor of City Hall.}
 {184}{}{Okay. And what's the other one?}
-{185}{}{There is an experienced pickpocket in Clean City. We received five applications for petty theft. That rascal needs to be caught as soon as possible. Be observant, you may be able to catch him in the act.}
+{185}{}{There is an experienced pickpocket in Clean City. We received five reports of petty theft. That rascal needs to be caught as soon as possible. If you're careful you may be able to catch him in the act.}
 {186}{}{Okay. The daughter of an archivist and a thief. I'll see what I can do.}
 
 {190}{}{What can you say about Anna Clark's disappearance?}
@@ -99,38 +99,38 @@
 {215}{}{One thousand bucks!}
 
 {216}{}{Very cunning of you. Good job! Underworld must be punished... And you should be rewarded. [Gives you $ 2000] As for those freaks - do not worry. We now arrange to round them up and they won't do any harm to anyone from now on.}
-{217}{}{I want to participate! I'm sick of my gun collecting dust from idleness.}
+{217}{}{I want to participate! I'm sick of my gun collecting dust.}
 {218}{}{Great!}
-{219}{}{Additional gun won't hurt.  I think the freaks' expulsion will be easy.}
-{220}{}{[You are waiting, while the police squad gets ready for a punitive operation to the Dead Hotel]}
+{219}{}{An additional gun won't hurt.  I think the freaks' expulsion will be easy.}
+{220}{}{[You are waiting, while the police squad gets ready for a punitive operation at the Dead Hotel]}
 
-{221}{}{I visited the freaks' hotel. They've got dope laying even in boxes! You should immediately check this!}
+{221}{}{I visited the freaks' hotel. They've got dope laying even in boxes! You should investigate immediately!}
 {222}{}{You are just in time. Thank you for the information, on behalf of the Las Vegas police. You are entitled to a reward. Here's 1,000$. As for the freaks - you can rest assured. We are now getting ready to go on a punitive operation.}
 
 #Улики против Уоллиса
 {223}{}{Do you know that Hal Wallace is involved in dirty business? I have proof!}
-{224}{}{Hal Wallace ?! This great man was involved in criminal activities? Do you understand that you are responsible for your words right now?}
-{225}{}{This is a record of our conversation. He persuaded me to illegally snitch the freaks!}
-{226}{}{This is a record of our conversation. He hired me, to steal Bob Neil's gun!}
+{224}{}{Hal Wallace?! This great man was involved in criminal activities? Do you understand that you are responsible for your words right now?}
+{225}{}{Here is a record of our conversation. He persuaded me to illegally snitch on the freaks!}
+{226}{}{Here is a record of our conversation. He hired me to steal Bob Neil's gun!}
 {227}{}{If everything is so serious, perhaps, I should refuse to present further evidence.}
 
 {228}{}{I do not think that someone in the police will seriously blame Mr. Wallace in such actions. Quite the contrary. But I must give evidence to our boss. Thank you, citizen. You're now free to go.}
-{229}{}{Can my incognito be guaranteed?}
+{229}{}{Can my anonymity be guaranteed?}
 {230}{}{Okay.}
 {231}{}{Yes, of course.}
-{232}{}{Yes, it is a serious offense. Unfortunately, I cannot take any early action. I will give the evidence to our boss, and he will have to make a final decision. Thank you, citizen, for your assistance. We will take it from here.}
+{232}{}{Yes, it is a serious offense. Unfortunately, I cannot take any immediate action. I will give the evidence to our boss, and he will have to make a final decision. Thank you, citizen, for your assistance. We will take it from here.}
 {233}{}{Don't I deserve some sort of a reward?}
 {234}{}{What do you think, why Wallace needed the pistol?}
-{235}{}{I hope, your chief will figure it out.}
+{235}{}{I hope your chief will figure it out.}
 {236}{}{No.}
 {237}{}{WHAT?! I risked my life and all was in vain! Fucking cop! Shove that evidence up your ass!}
 {238}{}{Shame.}
-{239}{}{I cannot say for sure. Perhaps he wanted set up Bob Neil and send him to prison. But why would he need that - I cannot understand.}
+{239}{}{I cannot say for sure. Perhaps he wanted set up Bob Neil and send him to prison. I have no idea why would he want that.}
 {240}{}{Right.}
 
 #Лаборатория Вегаса
 {241}{}{About the drugs that flood Vegas.}
-{242}{}{Well, well, well! What do you want to tell?}
+{242}{}{Well, well, well! What do you have to tell?}
 {243}{}{I know who supplies drugs to the city. They are called "black caravan".}
 {244}{}{I know where the laboratory is and I can show it on the map.}
 {245}{}{Uh... I see, it's an important matter to you. But I just wanted to ask how things are going.}
@@ -139,10 +139,10 @@
 {247}{}{Bad police work, officer!}
 {248}{}{Too bad.}
 
-{249}{}{*Very* good! Now we got these scum. Well, I thank you from behalf of the Clean City! You did the right thing by informing us about this. Traditionally, you deserve a reward. Here, this is for you.}
+{249}{}{*Very* good! Now we got these scum. Well, I thank you on behalf of the Clean City! You did the right thing by informing us about this. Traditionally, you deserve a reward. Here, this is for you.}
 {250}{}{Now that's a reward! Oh yes!}
 
-{251}{}{Now we got 'em! The Black Caravan will soon be of the color of ashes. Ha! Who would have thought, that in our times there is such honorable people left, like you. Here, take this reward. The whole Las Vegas is thankful to you. I hope you will continue to inform us about such criminal individuals.}
+{251}{}{Now we got 'em! The Black Caravan will soon be reduced to ashes. Ha! Who would have thought that in our times there is such honorable people left, like you. Here, take this reward. The whole Las Vegas is thankful to you. I hope you will continue to inform us about such criminal individuals.}
 {252}{}{Now that's a reward.}
 
 {253}{}{Can I count on the reward for the disclosure of the laboratory?}
@@ -153,7 +153,7 @@
 {258}{}{We rarely get help in such difficult matters... I think we could reward you, but what do you want?}
 {259}{}{To train my shooting skills.}
 {260}{}{Nothing, sorry.}
-{261}{}{Well, it is easy to arrange. But I think this will be our only concession. So don't haste with the choice.}
+{261}{}{Well, it is easy to arrange. But I think this will be a one time thing. So don't choose too hastily.}
 {262}{}{Small weapons.}
 {263}{}{Big weapons.}
 {264}{}{Energy weapons.}
@@ -176,13 +176,13 @@
 {279}{}{It was Orris, one of the Kings gang. He wanted to shut his mouth, just in case. I don't interfere in their affairs, I only follow orders.}
 
 {280}{}{[Hesitatingly] Really?}
-{281}{}{It's the truth, chief! That bastard deserved to die. He's one of the bandits, outside the walls of this city they make such crimes, that you never imagined of! And I... [sad] I had personal reasons.}
+{281}{}{It's the truth, chief! That bastard deserved to die. He's one of the bandits, outside the walls of this city they commit such crimes, that you never imagined of! And I... [sad] I had personal reasons.}
 {282}{}{I... Uh... Look, I understand you well. But you shouldn't have done this. I will get in trouble because of you! What should I tell to the authorities? What if someone comes in now?}
 {283}{}{I can just run, and you will say, that you didn't have the time to react! It is believable, no one will care about the bastard...}
-{284}{}{Hrmm... Ah fuck all to hell... Just run, before I change my mind! Now!}
+{284}{}{Hrmm... Ah fuck it all to hell... Just run, before I change my mind! Now!}
 {285}{}{Thanks, chief! Thanks!}
-{286}{}{Wait, I remember you! I heard about you, you fucking scum. And I almost believed you... [puts out the gun]}
-{287}{}{I knew it, that Kings will get involved. Fucking bastards! But you... Wait, you said that you're just a mercenary. Listen, I can let you go. But you'll have to do something to redeem yourself - kill Orris. That will be fair. If you won't do that, then you will get a bullet in your head as soon as you show yourself in Clean City. Got it? Now get the fuck out of here, before I change my mind!}
+{286}{}{Wait, I remember you! I heard about you, you fucking scum. And I almost believed you... [unholsters his gun]}
+{287}{}{I knew the Kings were involved! Fucking bastards! But you... Wait, you said that you're just a mercenary. Listen, I can let you go. But you'll have to do something to redeem yourself - kill Orris. That will be fair. If you won't do that, then you will get a bullet in your head as soon as you show yourself in Clean City. Got it? Now get the fuck out of here, before I change my mind!}
 {288}{}{[Leave]}
 {289}{}{Oh no! [Attack]}
 
